SignalRClient/WebsocketsTransport.swift

```diff
@@ -62,6 +66,12 @@ public class WebsocketsTransport: Transport {
 
         return url
     }
+    
+    @inline(__always) private func populateHeaders(headers: [String : String], request: inout URLRequest) {
```

Comment:
@inline keyword might be unnecessary since swift compiler will most likely inline this function anyways